ADR/adr-0007-change-management.md

```diff
@@ -0,0 +1,70 @@
+
+In Github, if the developer wants a code review but feels that their work is not ready to be deployed yet, they will add the "do-not-merge/work-in-progress" label to the PR. We also recommend adding "WIP" to the pull request title.
+
+The developer should choose one or two reviewers for their PR and add them to the PR directly. A PR without any reviewers assigned might never be reviewed. Reviewers should be chosen based on their background and familiarity with the code under review. It is also sometimes worthwhile to use a PR review to help another developer get up to speed with some code.
+
+If you have been asked to review another developer’s PR, it’s good practice to decide if you can fit that review into your current workload. Timely PR review helps keep our velocity high, and if it might take more than a couple of days for you to review the code, contact the PR author and let them know they might need to depend on another person.
+
+As a reviewer, you should move the review along by testing the PR yourself, and providing comments in Github for suggested changes. Typically we want to focus on potential security or privacy issues, test automation, maintainability factors and overall correctness of the code. Stylistic or preference-based changes should not be a focus of the review.
+
+We also require that all of our repositories be instrumented with CI test automation that runs when a PR is raised, and after each new commit. All CI checks must pass before performing a Merge to the main branch as this updates the staging environment and makes the code available for deployment into production.
+
+When the review is completed and tests have passed, the approver should Approve the PR in GitHub and the PR author will then Merge the code. 
+
+For changes to the infra-deployments repo, we recommend that the PR author add the "lgtm" (looks good to me) label to the PR rather than clicking on the Merge button. This will trigger the Prow/Tide deployment automation to create a batch of approved changes and test and deploy them together, usually within an hour. Batching changes likes this avoids merge race conditions.
+
+The ticket should be set to Closed once the code is released downstream or deployed to production.
+
+### When is a formal change request required?
+There are a few cases where we need to use Red Hat's formal Change Enablement process:
+* If your work can or will result in a production service becoming unavailable or degraded during service hours.
+* If the functionality of the service is changing.
+* If you are releasing new software, or a new major version of existing software.
+* If you are updating firmware or applying patches to existing infrastructure.
+
+This is not meant to be a complete list.  Most activities that impact production environments require a change request to be filed.  
```

Comment:
One example from the Quay team: "for any operations that affect push/pull of images, we give advance notice to customers 2 weeks before the planned change".  The use the formal change request process for that.